.github/workflows/build-pr-linux.yaml

```diff
@@ -0,0 +1,138 @@
+
+#
+# This source file is part of appleseed.
+# Visit https://appleseedhq.net/ for additional information and resources.
+#
+# This software is released under the MIT license.
+#
+# Copyright (c) 2022 Lars Zawallich, The appleseedhq Organization
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+name: Build and test on Ubuntu
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    branches: [master]
+
+jobs:
+  build-and-test-linux:
+    name: Linux
+    runs-on: ${{ matrix.os.version }}
+    strategy:
+      matrix:
+        os: [{name: Linux, version: ubuntu-18.04}]
+        compiler: 
+          - {CC: gcc-6, CXX: g++-6}
+          - {CC: gcc-7, CXX: g++-7}
+          - {CC: clang-8, CXX: clang++-8}
+        build_type: [Debug]
+
+    env: 
+      APPLESEED_DEPENDENCIES: $GITHUB_WORKSPACE/prebuilt-linux-deps
```

Comment:
There is a link to the prebuild linux deps here:
https://github.com/appleseedhq/appleseed/wiki/Building-appleseed-on-Linux
Since we host them in a Github Repo, we don't need to configure anything. Only if the deps change, we need to update them and then also update the `wget` command in here.